### DISCUSSION TITLE
Linking Ganache to Metamusk

### DISCUSSION THREAD

**Discussion by Skid-coder** [ConsenSys-archive/ganache#4549] [Q&A] [answered]
I tried creating a new network on my metamusk using ganache . I typed in the url "http://127.0.0.1:7545" and with some chain id 5777(specified on the ganache software)  which did not work and also tried out 1337 which did not work also. I want to know what the problem was. What is the correct url and the correct Chain id?

**Comment by davidmurdoch** [comment, marked as answer]
Are you using the npm package for ganache or the GUI?

In either case, try setting the chain id to 1337 everywhere (ganache chain id, network id, and in metamask).

**Reply by Skid-coder** [reply]
Thanks so much. It worked
